experimental/PropertyDDS/packages/property-changeset/src/test/schemaValidator.ts

```diff
@@ -57,13 +58,13 @@ export class SchemaValidator {
 	}
 
 	hasSchemaAsync = async (typeid) =>
-		new Promise(function (resolve, reject) {
-			setTimeout(function () {
+		new Promise((resolve, reject) => {
+			setTimeout(() => {
 				resolve(this.schemaMap[typeid] !== undefined);
 			}, 5);
 		});
```

Comment:
This code was bogus before as `this` wasn't captured. Fix is easy enough; so, fix. However, this code is never executed. Hence the invalid code wasn't a problem before.